regparser/commands/versions.py

```diff
@@ -17,12 +17,13 @@ def fetch_version_ids(cfr_title, cfr_part, notice_dir):
     register and the local filesystem"""
     version_ids = []
     final_rules = fetch_notice_json(cfr_title, cfr_part, only_final=True)
+    notice_ids = [v.path[-1] for v in notice_dir.sub_entries()]
 
     for document_number in (fr['document_number'] for fr in final_rules):
         # Document number followed by a date
         regex = re.compile(re.escape(document_number) + r"_\d{8}")
         version_ids.extend(
```

Comment:
Similar to https://github.com/eregs/regulations-parser/pull/297/commits/7a700663332e5c19616e07729395065c7dd8346d#r74863444 I tend to prefer breaking comprehensions up into chunks if they're longer than a line.